recipe/meta.yaml

```
@@ -40,6 +40,9 @@ requirements:
     - scipy >=1.8  # [aarch64]
     - joblib >=1.2.0
     - threadpoolctl >=3.1.0
+   # See https://github.com/conda-forge/openmp-feedstock/issues/126
+    - llvm-openmp  # [osx]
+    - libgomp  # [linux]
```

Do we need to do something like this (though maybe with tweaks) to better explicitly align versions between `build` & `host`?

```suggestion
   # See https://github.com/conda-forge/openmp-feedstock/issues/126
    - llvm-openmp  {{ llvm_openmp }} # [osx]
    - libgomp {{ libgomp }}  # [linux]
```

---

No, they are already pinned in `.ci_support`: https://github.com/conda-forge/scikit-learn-feedstock/blob/8bb0be319b176c2e16b48b7ca0f096fa5d17304c/.ci_support/osx_64_numpy2.0python3.11.____cpython.yaml#L21-L22

---

Is that sufficiently pinned for our needs?

---

Yes!

---

Sorry. Asking because this is only the major version and wasn't sure if that is enough for the use case that you asked about. If it is, great!